wf-core-tck-runner/rest-tck/pom.xml

```diff
@@ -197,26 +197,19 @@
                             <goal>run</goal>
                         </goals>
                     </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.wildfly.plugins</groupId>
-                <artifactId>wildfly-maven-plugin</artifactId>
-                <version>${wildfly.plugin.version}</version>
-                <executions>
                     <execution>
-                        <id>configure-wildfly</id>
+                        <id>enable-yaml-configuration</id>
                         <phase>process-test-classes</phase>
                         <goals>
-                            <goal>execute-commands</goal>
+                            <goal>run</goal>
                         </goals>
                         <configuration>
-                            <jboss-home>${jboss.home}</jboss-home>
-                            <offline>true</offline>
-                            <stdout>target/cli.out</stdout>
-                            <scripts>
-                                <script>src/test/resources/configure-wildfly.cli</script>
-                            </scripts>
+                            <target>
+                                <mkdir dir="${jboss.home}/modules/system/layers/base/org/jboss/as/controller/main/dir/META-INF/services/"/>
+                                <copy verbose="true" todir="${jboss.home}/modules/system/layers/base/org/jboss/as/controller/main/dir/META-INF/services/">
```

Comment:
Note to self: the 'overwrite' attribute defaults to 'false' which is good; we don't want to overwrite the file if it exists, which is does in newer code.